index.html

```diff
@@ -2229,7 +2238,96 @@ <h3>Global State</h3>
  configured. The default value of the flag is true if the endpoint
  doesn't support proxy configuration, or false otherwise.</p>
 
-<p>When asked to <dfn>close the session</dfn>,
+<div>
+
+<p>To <dfn>create a session</dfn>, given a JSON
+Object <var>capabilites</var>, and a set <var>flags</var>:
+<ol>
+ <li><p>Let <var>session id</var> be the result of <a>generating a UUID</a>.
+
+ <li><p>Let <var>session</var> be a new <a>session</a> with <a>session
+  ID</a> <var>session id</var>, and <a>HTTP flag</a> <var>flags</var> contains
+  "<code>http</code>".
+
+ <li><p>Let <var>proxy</var> be the result of getting property
+     "<code>proxy</code>" from <var>capabilities</var> and run the
+     substeps of the first matching statement:
+
+       <dl class=switch>
+         <dt><var>proxy</var> is a <a>proxy configuration</a> object</dt>
+         <dd><p>Take implementation-defined steps to set the user agent proxy
+             using the extracted <var>proxy</var> configuration. If the
+             defined proxy cannot be configured return <a>error</a> with
+             <a>error code</a> <a>session not created</a>. Otherwise
+             set the <a>has proxy configuration</a> flag to true.</dd>
```

Comment:
What if the flag is already `true`? Shouldn't we return early? I don't think that we should override the proxy settings as already set by a formerly created session.